packages/app/src/app/pages/Sandbox/Editor/Workspace/Files/DirectoryEntry/Entry/EditIcons/index.js

```diff
@@ -36,45 +35,43 @@ function EditIcons({
 
   return (
     <div className={className}>
-      {(hovering || (window.__isTouch && active) || forceShow) && (
```

Comment:
Since hovering prop is removed, this entire condition is removed because it is handled above without consideration for `hovering`.